scripts/pylint_extensions_test.py

```diff
@@ -1143,3 +1143,76 @@ def func3():
 
         with checker_test_object.assertNoMessages():
             temp_file.close()
+
+
+class SingleNewlineAboveArgsCheckerTests(unittest.TestCase):
+
+    def test_checks_newline_above_docstring(self):
```

Comment:
Add more tests like:
- Newline above Args and Returns doesn't return an error
- 2 Newlines above args and returns does throw an error
- The word returns in a comment in code doesn't throw an error.

Reply:
Done